addons/velero/template/testgrid/k8s-docker.yaml

```diff
@@ -0,0 +1,11 @@
+kubernetes:
+  version: "latest"
+weave:
+  version: "latest"
+rook:
+  version: "latest"
```

Comment:
I don't believe rook is required here, but I'm not positive

Reply:
I would guess that you're right but leaving this one since it makes sense for these to be deployed together.